templates/taxi/driver_list.html

```diff
@@ -6,8 +6,14 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.id == user.id %}
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Code style: The URL identification parameter `name` should use `-` to split words, not `_`. So it should be `taxi:driver-detail` instead of `taxi:driver_detail`.